src/components/Projects.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div>
+        <div class="Project">
+            <h1 class="projectTitle">Weather Station</h1>
+            <p class="projectText">This is our <a href="https://weather.codihacks.org/" target="_blank">weather station</a> web app project! It features a physical weather station, an API server
+                written in Rust, and a web app frontend written in Vue.
+            </p>
+        </div> 
+        <div class="Project">
+            <h1 class="projectTitle">Space Race</h1>
+            <p class="projectText">This is our Spaceship Race project! It is a spaceship game created with the Lua framework <a href="https://love2d.org/" target="_blank">Love2D</a>.
+                It lacks levels, so if you are interested in helping, please open an issue on our GitHub.
+            </p>
+            
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'Projects',
+    props: {
+    darkMode: {
+      type: Boolean,
+      required: true
+    }
+  }
+
+}
+</script>
+
+<style>
+        
+    .Project{
+        display: inline-block;
+        background-color: #89c7d0;
+        border-radius: 15px;
+        color: white;
+        height: auto;
+        width: 45vw;
+        max-width: 97%;
+        margin-top: 10vh;
+        margin-right: 1vw;
+        margin-left: 1vw;
+        margin-bottom: 20vh;
```

Comment:
Creating spacing between the Projects section and Footer section ought to be done on the parent element or parent's parent. Otherwise this puts a large gap between each row of projects (say if you had 3 or more).